File: baekjoon/준비운동 Part1. 튼튼한 기본기/1978_소수 찾기.py
n = int(input())
nums = list(map(int, input().split()))
count = 0

def isPrime(n):
    if n <= 1:
        return False
    for i in range(2, n-1):
        if n%i == 0:
            return False
    return True

for num in nums:
    if isPrime(num):
        count += 1

print(count)

File: baekjoon/알고리즘기초1/수학1/1978_소수찾기.js
const fs = require('fs');
const input = fs.readFileSync("../../test.txt").toString().trim().split('\n');

const N = Number(input.shift());
const numbers = input.shift().split(" ").map((el) => parseInt(el, 10));
let result = 0;

const isPrime = (num) => {
    if (num === 0 || num === 1) {
        return false
    }
    for (let i = 2; i < Math.floor(num); i++) {
        if (num % i === 0) {
            return false
        }
    }
    return true
}

for (let i = 0; i < N; i++) {
    if (isPrime(numbers[i])) {
        result += 1;
    }
}
console.log(result);